src/config/Config.ts

```diff
@@ -126,20 +126,20 @@ const conf = {
   }
 };
 
-export const config = () => {
-  return deepFreeze(conf) as typeof conf;
-};
+export const config = deepFreeze(conf) as typeof conf;
```

Comment:
Note about that: I don't know if there was a good reason to have this lazy `deepFreeze`, repeated for each access. Testing locally, it doesn't seem to break anything to set it once for all. Anyone has an idea about the reason? (@israel-hdez , @mtho11  maybe? )

Reply:
Not sure. `conf` is also a constant, so it shouldn't mutate. For me, it is better with your change.